### ISSUE TITLE
Client Dependency Framework (CDF) - Space in DesktopModule folder (module.css)

### ISSUE THREAD

**Issue by NerdyParker** [dnnsoftware/Dnn.Platform#4719]
## Description of bug
When a desktop module has a space in the file path (ie. /DesktopModules/Test Module/), it appears that the Client Dependency Framework (CDF) is adding a "%20" to the module.css path causing the module.css not to be included int he composite CSS. This is only an issue when "Enable Composite Files" is enabled in Site Settings.

It appears to be treating the path as a URL rather than a directory. You can confirm this further by manually creating a folder with the "%20" included. For example, add a module.css to the following folder: "/DesktopModules/Test%20Module/" and it is included in the composite CSS.

## Steps to reproduce
1. Create a new desktop module with a space in the directory name
2. Add a module.css file
3. Enable "Enable Composite Files" in Servers > Server Settings > Performance
3. The module.css file will not be included. Furthermore, you will see the following in the HTML comment in the header, confirming that a "%20" is added instead of a space:
`<!--CDF(Css|/DesktopModules/Test%20Module/module.css|DnnPageHeaderProvider|10)-->`

## Current behavior
If a desktop module has a space in its folder name, the module.css is not included in the composite CSS.

## Expected behavior
The module.css should be included in the composite CSS. The included path should not have a "%20" but rather a space.

## Screenshots
![image](https://user-images.githubusercontent.com/5623435/123143278-63053c80-d428-11eb-89e4-6ed7386aa355.png)

## Additional context
Provide any additional context that may be helpful in understanding and/or resolving the bug.

## Affected version
* [ ] 10.00.00 alpha build 
* [X] 09.09.00 latest supported release

## Affected browser
* [X] Chrome
* [X] Firefox
* [X] Safari
* [X] Internet Explorer 11
* [X] Microsoft Edge (Classic)
* [X] Microsoft Edge Chromium

**Comment by mitchelsellers** [comment]
This is an interesting one and I'm curious to others' opinions on this.  Spaces in folder names are a REALLY bad idea for web applications.

**Comment by NerdyParker** [comment]
Agree. However, this was previously working. I noticed the issue after upgrading from 9.1.1 to 9.3.2. I know that there are some modules that do indeed have a space in their folder (eg. Ventrian News Articles)

**Comment by bdukes** [comment]
My opinion is that it should work :-)

**Comment by valadas** [comment]
Yeah, I would never use a space in any folder name or file name, but agree it is a bug unless proven otherwise :)
